tools/generateVendorExportsFile.js

```diff
@@ -0,0 +1,54 @@
+const { promises: fs } = require("fs");
```

Comment:
This script reads the contents of the `vendors/` folder and generates a `src/vendors.js` that contains the list of all vendors.